app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
Consider adding `wife` and `husband` attributes to the `Person` class. These attributes are necessary for setting spouse relationships in the `create_person_list` function.